cantusdb/turtle/csv2rdf_by_RDFLib.py

```diff
@@ -0,0 +1,51 @@
+from rdflib import Graph, URIRef, Literal
+from rdflib.namespace import RDF
+import csv
+import validators
+
+# Define the base and prefix URIs
+base_uri = "http://www.semanticweb.org/LinkedMusic/ontologies/"
+prefix_uri = {
+    "wdt": "http://www.wikidata.org/prop/direct/",
+    "cantusDB": "https://cantusdatabase.org/",
+}
+
+def convert_csv_to_turtle(csv_filename):
+    with open(csv_filename, 'r', encoding='utf-8') as csv_file:
+        g = Graph()
+        csv_reader = csv.DictReader(csv_file)
+        
+        chant_type = URIRef(prefix_uri["cantusDB"] + "chant")
+        
+        # Add predicate URIs
+        p_incipit = URIRef(prefix_uri["wdt"] + "P1922")
```

Comment:
Could these be constants defined outside of the function? Also, you could use f-strings here:

```python
P_INCIPIT = URIRef(f"{prefix_uri['wdt']}P1922")
```